rubric_data/tests.py

```diff
@@ -56,84 +62,29 @@ def test_datapoints_list(self):
         self.assertEqual(denormalized_data[6]["score"], 2)
         self.assertEqual(denormalized_data[6]["rating"], "Not Good enough")
 
+    def test_datapoints_list_raises_error(self):
+        rubric_assignments = rubric_assignments_dict(MOCK_ASSIGNMENTS_LIST)
+        students = students_sections_tuple(MOCK_SECTIONS_LIST)[0]
+        criteria_lookup = { criterion["id"]: criterion
+            for assignment in rubric_assignments
+            for criterion in rubric_assignments[assignment]["rubric"]
+        }
+        
+        with self.assertRaises(DatapointsError) as context:
+            datapoints_list(criteria_lookup, students, [])
+        self.assertEqual("submissions list is empty.", context.exception.message)
+
     # https://docs.djangoproject.com/en/3.2/topics/cache/#local-memory-caching
     # https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.override_settings
     # override_settings is used to override the cache to use local memory instead of redis for testing
     @override_settings(CACHES = {'default': {'BACKEND': 'django.core.cache.backends.locmem.LocMemCache'}})
     def test_create_cache(self):
-        user_id = uuid.uuid4().hex
-        course_id = random.randint(10000, 99999)
+        user_id = 'ad12732ghs'
+        course_id = 1112354
```

Comment:
Just curious, but why the switch back to hard-coded IDs? Were the random IDs causing problems?

Reply:
Not causing problems, but the tests function just fine with static values. Removing those calls let us remove the `uuid` and `random` library imports. Just a simple refactoring.